Rakefile

```diff
@@ -103,6 +105,16 @@ task :lint do
   sh "bundle exec pod lib lint --allow-warnings"
 end
 
+task :take_screenshots do
+  # Remove existing screenshots
+  FileUtils.rm_rf(Dir.glob('screenshots/*'))
```

Comment:
Do we have to glob here? `rm_rf` already removes recursively so

```ruby
FileUtils.rm_rf('screenshots')
```

should be enough

Reply:
But we don't want to delete `screenshots`, just everything inside it. Hence the `glob` and the `*`

Reply:
Isn't everything in `screenshots` generated though? Seems cleaner to just remove the whole thing

Reply:
Actually it still works. I guess `fastlane` recreates the directory so 🤷‍♀️